docs/frontend-system/building-plugins/03-common-extension-blueprints.md

```diff
@@ -46,18 +46,26 @@ Icon bundle extensions provide the ability to replace or provide new icons to th
 
 Translation extension provide custom translation messages for the app. They can be used both to override the default english messages to custom ones, as well as provide translations for additional languages.
 
-## Core feature extension blueprints
+## Extension blueprints in `@backstage/plugin-catalog-react/alpha`
 
-These are the [extension blueprints](../architecture/23-extension-blueprints.md) provided by the Backstage core feature plugins.
+These are the [extension blueprints](../architecture/23-extension-blueprints.md) provided by the Catalog plugin.
 
-### EntityCard - [Reference](https://github.com/backstage/backstage/blob/master/plugins/catalog-react/report-alpha.api.md)
+### EntityCard - [Example](https://github.com/backstage/backstage/blob/75e79518eafc6e6eb55585f166667418419662de/plugins/org/src/alpha.tsx#L27-L36)
```

Comment:
Best I could find so far, we need better references here. Ideally we could completely skip this section and rely on some other auto generated documentation though.